documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -96,6 +96,41 @@ spec:
 .Additional resources
 For more information, refer to the xref:type-BrokerCapacity-reference[].
 
+== Topic creation and configuration
+
+Cruise Control requires the following three topics to be created in order to function properly:
+
+* `strimzi.cruisecontrol.partitionmetricsamples`
+* `strimzi.cruisecontrol.modeltrainingsamples`
+* `strimzi.cruisecontrol.metrics`
+
+Cruise Control will attempt to create these topics automatically.
+However, if automatic topic creation is disabled in Kafka with a configuration of `auto.create.topics.enable: false` in `spec.kafka.config`, Cruise Control will be unable to create these topics.
```

Comment:
This is confusing. `auto.create.topics.enable` controls whether the broker creates a topic when a client attempts to read or write from it. So if it cannot create the topic when `auto.create.topics.enable: false`, it sounds like CC will not really attempt to create the topic and relies only on the auto creation. So the first sentence is wrong.

Reply:
Updated the wording